subprojects/docs/src/docs/userguide/more_about_tasks.adoc

```diff
@@ -473,6 +473,23 @@ include::{samplesPath}/userguide/tutorial/disableTask/build.gradle[]
 include::{samplesPath}/userguide/tutorial/disableTask/disableTask.out[]
 ----
 
+[[sec:task_timeouts]]
+=== Task timeouts
+
+Every task has a `timeout` property which can be used to limit its execution time.
+When a task reaches its timeout, its task execution thread is interrupted.
+The task will be marked as failed. Finalizer tasks will still be run.
+If `--continue` is used, other tasks can continue running after it.
+Tasks that don't respond to interrupts can't be timed out.
+All of Gradle's built-in tasks respond to timeouts in a timely manner.
+
+=== Example: Specifying task timeouts
+
+====
```

Comment:
Doesn't this have to use `[source.multi-language-sample,groovy]`?

Reply:
> Doesn't this have to use [source.multi-language-sample,groovy]?

Nope! The Asciidoctor extension automatically adds this based on the file suffix :)